File: redsheet-portal/src/app/database-models/v-our-team-member.ts
import { required, maxLength, range, nested } from '@rx/annotations';

export class vOurTeamMember {
    constructor(vOurTeamMember?: vOurTeamMember )  {
        let properties = [ "agreeableBehaviour", "agreeableRequire", "assertivenessBehaviour", "assertivenessChange", "assertivenessRequire", "changeAgreeable", "changeConsciousness", "changeOpenMindedId", "changeOuting", "changePersonalCalm", "changeSolutionFocused", "changeWIllToWin", "conflictStyleBehaviour", "conflictStyleChange", "conflictStyleRequire", "consciousnessBehaviour", "consciousnessRequire", "emotionalCompetenceBehaviour", "emotionalCompetenceChange", "emotionalCompetenceRequire", "openMindedBehaviour", "openMindedRequire", "ourTeamMemberId", "ourTeamMemberName", "outingBehaviour", "outingRequire", "personalCalmBehaviour", "personalCalmRequire", "projectNegotionalityId", "solutionFocusedBehaviour", "solutionFocusedRequire", "teamRoleId", "teamRoleName", "userId", "wIllToWinBehaviour", "willToWinRequire",];
        for (let property of properties)
            if (vOurTeamMember && vOurTeamMember[property])
                this[property] = vOurTeamMember[property];
    }
 
    @maxLength(1)
	agreeableBehaviour : string =   undefined;
 
    @maxLength(1)
	agreeableRequire : string =   undefined;
 
    @maxLength(50)
	assertivenessBehaviour : string =   undefined;
 
    @maxLength(50)
	assertivenessChange : string =   undefined;
 
    @maxLength(50)
	assertivenessRequire : string =   undefined;
 
	changeAgreeable : number =   undefined;
 
	changeConsciousness : number =   undefined;
 
	changeOpenMindedId : number =   undefined;
 
	changeOuting : number =   undefined;
 
	changePersonalCalm : number =   undefined;
 
	changeSolutionFocused : number =   undefined;
 
	changeWIllToWin : number =   undefined;
 
    @maxLength(50)
	conflictStyleBehaviour : string =   undefined;
 
    @maxLength(50)
	conflictStyleChange : string =   undefined;
 
    @maxLength(50)
	conflictStyleRequire : string =   undefined;
 
    @maxLength(1)
	consciousnessBehaviour : string =   undefined;
 
    @maxLength(1)
	consciousnessRequire : string =   undefined;
 
    @maxLength(50)
	emotionalCompetenceBehaviour : string =   undefined;
 
    @maxLength(50)
	emotionalCompetenceChange : string =   undefined;
 
    @maxLength(50)
	emotionalCompetenceRequire : string =   undefined;
 
    @maxLength(1)
	openMindedBehaviour : string =   undefined;
 
    @maxLength(1)
	openMindedRequire : string =   undefined;
 
	ourTeamMemberId : number =   0 ;
 
    @required()
    @maxLength(201)
	ourTeamMemberName : string =   undefined;
 
    @maxLength(1)
	outingBehaviour : string =   undefined;
 
    @maxLength(1)
	outingRequire : string =   undefined;
 
    @maxLength(1)
	personalCalmBehaviour : string =   undefined;
 
    @maxLength(1)
	personalCalmRequire : string =   undefined;
 
    @range(1,2147483647)
	projectNegotionalityId : number =   undefined;
 
    @maxLength(1)
	solutionFocusedBehaviour : string =   undefined;
 
    @maxLength(1)
	solutionFocusedRequire : string =   undefined;
 
    @range(1,2147483647)
	teamRoleId : number =   undefined;
 
    @required()
    @maxLength(200)
	teamRoleName : string =   undefined;
 
    @range(1,2147483647)
	userId : number =   undefined;
 
    @maxLength(1)
	wIllToWinBehaviour : string =   undefined;
 
    @maxLength(1)
	willToWinRequire : string =   undefined;


}

File: RedSheet.DbEntities/Models/vOurTeamMember.cs

using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.ComponentModel.DataAnnotations;
using Microsoft.EntityFrameworkCore;
using Rx.Core.Data.Attributes;
using RedSheet.DbEntities.Enums;

namespace RedSheet.DbEntities.Models
{
	[Table("vOurTeamMembers", Schema = "dbo")]
	[Serializable]
	public partial class vOurTeamMember  
	{
 
	[MaxLength(1)]
	public string AgreeableBehaviour { get; set; } 
 
	[MaxLength(1)]
	public string AgreeableRequire { get; set; } 
 
	[MaxLength(50)]
	public string AssertivenessBehaviour { get; set; } 
 
	[MaxLength(50)]
	public string AssertivenessChange { get; set; } 
 
	[MaxLength(50)]
	public string AssertivenessRequire { get; set; } 

	public Nullable<int> ChangeAgreeable { get; set; } 

	public Nullable<int> ChangeConsciousness { get; set; } 

	public Nullable<int> ChangeOpenMindedId { get; set; } 

	public Nullable<int> ChangeOuting { get; set; } 

	public Nullable<int> ChangePersonalCalm { get; set; } 

	public Nullable<int> ChangeSolutionFocused { get; set; } 

	public Nullable<int> ChangeWIllToWin { get; set; } 
 
	[MaxLength(50)]
	public string ConflictStyleBehaviour { get; set; } 
 
	[MaxLength(50)]
	public string ConflictStyleChange { get; set; } 
 
	[MaxLength(50)]
	public string ConflictStyleRequire { get; set; } 
 
	[MaxLength(1)]
	public string ConsciousnessBehaviour { get; set; } 
 
	[MaxLength(1)]
	public string ConsciousnessRequire { get; set; } 
 
	[MaxLength(50)]
	public string EmotionalCompetenceBehaviour { get; set; } 
 
	[MaxLength(50)]
	public string EmotionalCompetenceChange { get; set; } 
 
	[MaxLength(50)]
	public string EmotionalCompetenceRequire { get; set; } 
 
	[MaxLength(1)]
	public string OpenMindedBehaviour { get; set; } 
 
	[MaxLength(1)]
	public string OpenMindedRequire { get; set; } 
 
	[DatabaseGenerated(DatabaseGeneratedOption.None)] 
	[Key]
	public int OurTeamMemberId { get; set; } 
 
	[Required] 
	[MaxLength(201)]
	public string OurTeamMemberName { get; set; } 
 
	[MaxLength(1)]
	public string OutingBehaviour { get; set; } 
 
	[MaxLength(1)]
	public string OutingRequire { get; set; } 
 
	[MaxLength(1)]
	public string PersonalCalmBehaviour { get; set; } 
 
	[MaxLength(1)]
	public string PersonalCalmRequire { get; set; } 
 
	[Range(1, int.MaxValue)]
	public int ProjectNegotionalityId { get; set; } 
 
	[MaxLength(1)]
	public string SolutionFocusedBehaviour { get; set; } 
 
	[MaxLength(1)]
	public string SolutionFocusedRequire { get; set; } 
 
	[Range(1, int.MaxValue)]
	public int TeamRoleId { get; set; } 
 
	[Required] 
	[MaxLength(200)]
	public string TeamRoleName { get; set; } 
 
	[Range(1, int.MaxValue)]
	public int UserId { get; set; } 
 
	[MaxLength(1)]
	public string WIllToWinBehaviour { get; set; } 
 
	[MaxLength(1)]
	public string WillToWinRequire { get; set; } 

	public vOurTeamMember()
	{
    }

    }
}

